### PR TITLE
[IMP] do some cleanup on lettermgmt

### PR DESCRIPTION
**Pull request by hbrunn**
for discussion, see #10

### CODE REVIEW

**Comment by hbrunn** [comment]
I merged #12 to avoid travis errors from lettermgmt_hr into this branch

**Comment by yvaucher** [comment]
:+1:

File: lettermgmt/models/letter_type.py [hunk 1]
```diff
@@ -20,18 +20,16 @@
...
     _name = 'letter.type'
-    _description = _("Letter Type")
+    _description = "Letter Type"
     _columns = {
-        'name': fields.char('Type', size=32, required=True),
-        'code': fields.char('Code', size=8, required=True),
+        'name': fields.char('Type', required=True),
+        'code': fields.char('Code', required=True),
     }
     _sql_constraints = [('code_uniq', 'unique(code)', 'Code must be unique !')]
```
Comment (ehdem) [inline lettermgmt/models/letter_type.py:35]: Add _('Code must be unique !') to be translate.

Comment (guewen) [inline lettermgmt/models/letter_type.py:35]: > Add _('Code must be unique !') to be translate.

I am sure this is not required, it is added to the translations by the ORM when it looks up the _sql_constraints.

**Comment by guewen** [comment]
The module needed your care, thanks
:+1:

**Comment by guewen** [comment]
oh, 2 pep8 errors, I will correct them when merging:

```
./lettermgmt/migrations/7.0.1.1/post-migrate.py:21:1: E302 expected 2 blank lines, found 0
./lettermgmt/migrations/7.0.1.1/pre-migrate.py:21:1: E302 expected 2 blank lines, found 0
```

**Pull request merged**